src/interpreter/plugin/FunctionPlugin.ts

```diff
@@ -42,7 +42,7 @@ export interface FunctionMetadata extends FunctionArguments{
   /**
    * Used for functions with variable number of arguments -- last defined argument is repeated indefinitely.
    */
-  repeatLastArg?: boolean,
+  repeatLastArgs?: number,
```

Comment:
`parameters`, `repeatLastArgs`, `expandRanges` are already inherited from `FunctionArguments` and can be removed from this interface.